### ISSUE TITLE
Codebuild pipeline fails in BUILD phase

### ISSUE THREAD

**Issue by andasv** [aws-solutions/automated-data-analytics-on-aws#31]
**Describe the bug**
Deployment of CF template fails, because Codebuild project fails, with the following error messages:

COMMAND_EXECUTION_ERROR: Error while executing command: aws s3 cp s3://$cloudAssemblyBucketName/automated-data-analytics-on-aws/v1.1.0/ada-cloud-assembly.zip .. Reason: exit status 1

AccessDeniedException: User: arn:aws:sts::xxx:assumed-role/myada-AdaOneClickBuildProjectRole71D42DAA-YGDBC8UJB25S/AWSCodeBuild-1fa0845c-2a5b-464a-bb60-2f101b1518c9 is not authorized to perform: logs:PutLogEvents on resource: arn:aws:logs:eu-central-1:xxx:log-group:/aws/codebuild/AdaOneClickBuildProjectB387-1AmywPeTLQFR:log-stream:1fa0845c-2a5b-464a-bb60-2f101b1518c9 because no identity-based policy allows the logs:PutLogEvents action status code: 400, request id: 0b4cfd59-acd4-46f9-9b4a-422e1ca689e9

**To Reproduce**
run CF deployment in eu-central-1

**Expected behavior**
CF deployment completes

**Please complete the following information about the solution:**
- Version: [most recent one at time of issuing this bug]

To get the version of the solution, you can look at the description of the created CloudFormation stack. For example, "_(SO0021) - Video On Demand workflow with AWS Step Functions, MediaConvert, MediaPackage, S3, CloudFront and DynamoDB. Version **v5.0.0**_". If the description does not contain the version information, you can look at the mappings section of the template:

```yaml
Mappings:
  SourceCode:
    General:
      S3Bucket: "solutions"
      KeyPrefix: "video-on-demand-on-aws/v5.0.0"
```

- [eu-central-1 ] Region: []
- [no ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [yes] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses?
- [ no] Were there any errors in the CloudWatch Logs?

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**

**Comment by rashimr** [comment]
Hey @andasv, thanks for your interest in the solution.

I am looking into this.

**Comment by rashimr** [comment]
Hey @andasv,

I have tried to recreate the issue;

1. I have signed into my AWS Console
2. I have run cdk bootstraop in eu-central-1 (cdk bootstrap aws://XXXXXXXXXXXX/eu-central-1)
3. From; https://aws.amazon.com/solutions/implementations/automated-data-analytics-on-aws/, I have clicked 'Launch in the AWS Console'
4. In my console I have set the region to eu-central-1.
5. I updated the template name, adminEmail and adminPhoneNumber to my values.
6. Leaving everything else as default i have launched the template.

The Codebuild deploys ADA as expected.

<img width="1209" alt="Screen Shot 2023-03-08 at 3 22 02 am" src="https://user-images.githubusercontent.com/100130038/223484058-3261ccac-4d0c-410e-bed1-3c7d7550c66b.png">

ada-testing being the name I entered for the template, and Ada being deployed from ada-testing codebuild.

Did you bootstrap your account and follow all the prerequisites outlined here: https://docs.aws.amazon.com/solutions/latest/automated-data-analytics-on-aws/automated-deployment.html#prerequisites

What are the specific steps you took to run the Cloudformation Template?

Thanks,
Rashim
